<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindows</class>
 <widget class="QMainWindow" name="MainWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTandOpencv</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>myfile/mydrawing.png</normaloff>myfile/mydrawing.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_left">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="zzzlyout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_right">
         <property name="minimumSize">
          <size>
           <width>410</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>410</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QScrollArea" name="scrollArea">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>404</width>
                  <height>548</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item row="2" column="0">
                  <widget class="QGroupBox" name="groupBox_textdisplay">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="title">
                    <string/>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_5">
                    <item row="0" column="0">
                     <widget class="QPlainTextEdit" name="textdisplay">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>350</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>11</pointsize>
                       </font>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::NoFrame</enum>
                      </property>
                      <property name="readOnly">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QGroupBox" name="groupBox_state">
                   <property name="minimumSize">
                    <size>
                     <width>350</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="title">
                    <string/>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_6">
                    <item row="0" column="0">
                     <layout class="QHBoxLayout" name="horizontalLayout">
                      <property name="spacing">
                       <number>19</number>
                      </property>
                      <item>
                       <widget class="QPushButton" name="btnPruduction">
                        <property name="minimumSize">
                         <size>
                          <width>115</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>16777215</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="font">
                         <font>
                          <pointsize>11</pointsize>
                         </font>
                        </property>
                        <property name="cursor">
                         <cursorShape>ArrowCursor</cursorShape>
                        </property>
                        <property name="text">
                         <string>生产统计</string>
                        </property>
                        <property name="icon">
                         <iconset>
                          <normaloff>myfile/images/Mabe/cart.jpg</normaloff>myfile/images/Mabe/cart.jpg</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                        <property name="checkable">
                         <bool>true</bool>
                        </property>
                        <property name="checked">
                         <bool>false</bool>
                        </property>
                        <property name="autoExclusive">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="btnCommunication">
                        <property name="minimumSize">
                         <size>
                          <width>115</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="font">
                         <font>
                          <pointsize>11</pointsize>
                         </font>
                        </property>
                        <property name="cursor">
                         <cursorShape>ArrowCursor</cursorShape>
                        </property>
                        <property name="mouseTracking">
                         <bool>false</bool>
                        </property>
                        <property name="text">
                         <string>通讯状态</string>
                        </property>
                        <property name="icon">
                         <iconset>
                          <normaloff>myfile/images/signal.jpg</normaloff>myfile/images/signal.jpg</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                        <property name="checkable">
                         <bool>true</bool>
                        </property>
                        <property name="autoExclusive">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="btnCammer">
                        <property name="minimumSize">
                         <size>
                          <width>115</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="font">
                         <font>
                          <pointsize>11</pointsize>
                         </font>
                        </property>
                        <property name="cursor">
                         <cursorShape>ArrowCursor</cursorShape>
                        </property>
                        <property name="text">
                         <string>相机状态</string>
                        </property>
                        <property name="icon">
                         <iconset>
                          <normaloff>myfile/images/camera.jpg</normaloff>myfile/images/camera.jpg</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>20</width>
                          <height>20</height>
                         </size>
                        </property>
                        <property name="checkable">
                         <bool>true</bool>
                        </property>
                        <property name="autoExclusive">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item row="1" column="0">
                     <widget class="QGroupBox" name="groupBox_Camer">
                      <property name="title">
                       <string/>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_7"/>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QGroupBox" name="groupBox_Comunication">
                      <property name="title">
                       <string/>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_8"/>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QGroupBox" name="groupBox_production">
                      <property name="title">
                       <string/>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_9"/>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QGroupBox" name="groupBox_data">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="title">
                    <string/>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_10">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item row="0" column="0">
                     <widget class="QGroupBox" name="groupBox_databtn">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>45</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>999999</width>
                        <height>45</height>
                       </size>
                      </property>
                      <property name="title">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QGroupBox" name="groupBox_datatable">
                      <property name="title">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="production">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>ArrowCursor</cursorShape>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::DefaultContextMenu</enum>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>生产状态</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>myfile/images/Mabe/cards.jpg</normaloff>myfile/images/Mabe/cards.jpg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="dayNoteBook">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>ArrowCursor</cursorShape>
              </property>
              <property name="text">
               <string>生产日志</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>myfile/images/Mabe/book-opened.jpg</normaloff>myfile/images/Mabe/book-opened.jpg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="data">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>ArrowCursor</cursorShape>
              </property>
              <property name="text">
               <string>生产数据</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>myfile/images/Mabe/chart.jpg</normaloff>myfile/images/Mabe/chart.jpg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1139</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="separator"/>
   <addaction name="actionUserSign"/>
   <addaction name="actionUseradmini"/>
   <addaction name="actionExitSign"/>
   <addaction name="separator"/>
   <addaction name="actionCommunication"/>
   <addaction name="actionAllUseDemo"/>
   <addaction name="actionSetting"/>
   <addaction name="actionTips"/>
  </widget>
  <action name="actionCommunication">
   <property name="icon">
    <iconset>
     <normaloff>myfile/images/signal.jpg</normaloff>myfile/images/signal.jpg</iconset>
   </property>
   <property name="text">
    <string>Communication</string>
   </property>
   <property name="iconText">
    <string>通讯管理</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionUserSign">
   <property name="icon">
    <iconset>
     <normaloff>myfile/images/person.jpg</normaloff>myfile/images/person.jpg</iconset>
   </property>
   <property name="text">
    <string>UserSign</string>
   </property>
   <property name="iconText">
    <string>用户登录</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionExitSign">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>myfile/images/skull.jpg</normaloff>myfile/images/skull.jpg</iconset>
   </property>
   <property name="text">
    <string>ExitSign</string>
   </property>
   <property name="iconText">
    <string>用户退出</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>myfile/images/power.jpg</normaloff>myfile/images/power.jpg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionUseradmini">
   <property name="icon">
    <iconset>
     <normaloff>myfile/images/people.jpg</normaloff>myfile/images/people.jpg</iconset>
   </property>
   <property name="text">
    <string>Useradmini</string>
   </property>
   <property name="iconText">
    <string>用户管理</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset>
     <normaloff>myfile/images/settings.jpg</normaloff>myfile/images/settings.jpg</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="iconText">
    <string>设置</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionTips">
   <property name="icon">
    <iconset>
     <normaloff>myfile/images/help.jpg</normaloff>myfile/images/help.jpg</iconset>
   </property>
   <property name="text">
    <string>Tips</string>
   </property>
   <property name="iconText">
    <string>提示</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAllUseDemo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>myfile/images/Mabe/anchor.jpg</normaloff>myfile/images/Mabe/anchor.jpg</iconset>
   </property>
   <property name="text">
    <string>AllUseDemo</string>
   </property>
   <property name="iconText">
    <string>压力测试</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="icon">
    <iconset>
     <normaloff>myfile/images/language.jpg</normaloff>myfile/images/language.jpg</iconset>
   </property>
   <property name="text">
    <string>Language</string>
   </property>
   <property name="iconText">
    <string>语言</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>myfile/images/play.jpg</normaloff>myfile/images/play.jpg</iconset>
   </property>
   <property name="text">
    <string>自动运行</string>
   </property>
   <property name="toolTip">
    <string>自动运行</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QTandOpencv.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindows</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_Usecv_clicked()</slot>
  <slot>on_Window_change()</slot>
  <slot>on_Why_clicked()</slot>
 </slots>
</ui>
